package.json

```diff
@@ -35,6 +35,7 @@
     "start": "node server/index.js",
     "server": "nodemon --watch client --watch server --watch templates --ext js,jsx server/index.js",
     "server-dev": "nodemon --watch client --watch server --watch templates --ext js,jsx server/index-dev.js",
+    "server-hot": "nodemon --watch client --watch server --watch templates --ext js,jsx server/index-hot.js",
```

Comment:
@alexlande -- This repo is also fully windows compatible. The usual `FOO=bar my-command.js` breaks windows.